packages/rocket-file-uploads-aws/src/handlers/delete-file.ts

```diff
@@ -0,0 +1,20 @@
+import { BoosterConfig } from '@boostercloud/framework-types'
+import { RocketFilesUserConfiguration } from '@boostercloud/rocket-file-uploads-types'
+import * as AWS from 'aws-sdk'
+
+export async function deleteFile(
+  config: BoosterConfig, 
+  rocketFilesUserConfiguration: RocketFilesUserConfiguration, 
+  directory: string, 
+  fileName: string
+): Promise<boolean> {
+  const s3 = new AWS.S3()
+  const bucketName = rocketFilesUserConfiguration.storageName + '-' + config.environmentName
+  const params = {
+    Bucket: bucketName,
+    Key: `${directory}/${fileName}`,
+  }
+  
+  const result = await s3.deleteObject(params).promise()
+  return result.$response.error == null
```

Comment:
`==` checks for reference equality, also, in JS world there are three different nullable values (`undefined`, `null` and non-existent), it is best to cast to a boolean value:
```suggestion
  return Boolean(result.$response.error) === false
```